.github/workflows/ubuntu.yml

```
@@ -0,0 +1,84 @@
+name: Ubuntu CI
+
+on:
+  push:
+    branches: [ "main" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)